libs/community/langchain_community/llms/oci_data_science_model_deployment_endpoint.py

```diff
@@ -256,33 +599,26 @@ def _llm_type(self) -> str:
     def _default_params(self) -> Dict[str, Any]:
         """Get the default parameters for invoking OCI model deployment TGI endpoint."""
         return {
+            "model": DEFAULT_MODEL_NAME,  # can be any
             "best_of": self.best_of,
             "max_new_tokens": self.max_tokens,
             "temperature": self.temperature,
-            "top_k": self.k
-            if self.k > 0
-            else None,  # `top_k` must be strictly positive'
+            "top_k": (
```

Comment:
Seems like top_k is not in the new TGI spec?

Reply:
As we discussed offline, we introducing `endpoint_spec` for incorporate both `/generate` and `/v1/completions` spec.